doc/nrf/known_issues.rst

```diff
@@ -19,7 +19,8 @@ Use the drop-down filter to see known issues for previous releases and check if
 
    <select name="versions" id="versions-select">
```

Comment:
@peknis , can you help me with an issue?
The lines 58-62 are not displayed on the live page: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/known_issues.html

I think the reason is the `.. raw::html`  section. Can you try adding a closing `..` in line 57 and add a blank line after it?

If that doesn't work and the lines are not displaying, can you move them to line 13?

Reply:
Fixed as discussed offline.